### DISCUSSION TITLE
[M-Z] (9 Settembre 2019) Esercizio 1

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#447] [Esami & Esoneri M-Z]
## Traccia

<img width="876" alt="image" src="https://github.com/user-attachments/assets/4cb920ba-6043-4a9d-b384-be084848ccc0">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190909.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190909.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/2ec44020-1a29-4e56-8e8c-021fc92b5c40)
